lib/split/experiment.rb

```diff
@@ -268,122 +268,6 @@ def load_from_redis
       set_alternatives_and_options(options)
     end
 
-    def calc_winning_alternatives
```

Comment:
This is the beginning of the callstack in which there are methods we no longer need (that only exist for calculating winner alternatives...which we are removing due to SimpleRandom).